### ISSUE TITLE
Unable to translate DBCompositeField

### ISSUE THREAD

**Issue by sanderha** [tractorcow/silverstripe-fluent#435]
*sorry if this is not the right place for this issue

I'm using the [Multivaluefield module ](https://github.com/symbiote/silverstripe-multivaluefield), which has created its own DB field type called `MultiValueField`. This class extends `DBComposite`.
Fluent does not translate this field, which it should, since it just contains a string.
How can we go about translating this type of field?

**Comment by robbieaverill** [comment]
Can you please provide some code to allow us to reproduce your problem?

**Comment by robbieaverill** [comment]
Please reopen when there's enough information to reproduce or help you out =)

**Comment by dhensby** [comment]
Hi, thanks for taking the time to post to our issue tracker.

The issue tracker is reserved for reporting bugs and we don't provide support via the issue tracker. There are great [community support options](http://www.silverstripe.org/community/#introduction) where you can get help including [our forum](https://forum.silverstripe.org), [Stack Overflow (remember to tag as silverstripe)](http://stackoverflow.com/questions/tagged/silverstripe) and even [IRC](http://www.silverstripe.org/community/contributing-to-silverstripe/irc-channel/) or [Slack](http://silverstripe.org/slack).

Sorry we can't be of more help and good luck resolving your problem!

**Comment by sanderha** [comment]
@robbieaverill @dhensby 
Isnt it a bug `DBComposite` cannot be translated? 

Its not possible to translate a `Money` field either I noticed. I suspect its because the actual columns in the DB get a different name, than the field name.

**Comment by dhensby** [comment]
@sanderha yes, sorry, it may be, can you provide some more information as @robbieaverill has requested? thanks

**Comment by sanderha** [comment]
Alright here's an example with data type `Money`.
My model has this:
```
    private static $db = [
        'CustomMoneyField' => 'Money'
    ];
```
I've then defined this for Fluent to translate it:
```
Page:
  translate:
    - 'CustomMoneyField'
```
Doing this, no fields at all are created in the Localised table.

If I do this instead:
```
Page:
  translate:
    - 'CustomMoneyFieldCurrency'
    - 'CustomMoneyFieldAmount'
```
Fluent creates the fields in the Localised table.

However, editing the field `CustomMoneyField` in the CMS, does not save to the correct Localised table, it just saved to the (non localised) default table. It also does not add the CSS classes to the field.

If I try to output `$CustomMoneyField` or `$CustomMoneyField.Currency` in the template, and switch to a language that is *not* the default locale, I get an `ERR_TOO_MANY_REDIRECTS` browser error.

Honestly I dont know where to look in the Fluent module to attempt to fix this.

**Comment by robbieaverill** [comment]
Can you please provide the versions of each module you're using?

**Comment by sanderha** [comment]
Yes sorry.
I'm using Fluent v 4.1.3 and silverstripe/recipe-cms 4.1.2
